client/settings/account-details/index.js

```diff
@@ -3,50 +3,69 @@ import { createInterpolateElement } from '@wordpress/element';
 import React from 'react';
 import SectionStatus from '../../components/section-status';
 import './style.scss';
+import { useAccount, useGetCapabilities } from 'wcstripe/data/account';
+
+const useIsCardPaymentsEnabled = () =>
+	useGetCapabilities().card_payments === 'active';
+
+const useAreDepositsEnabled = () => {
+	const { data } = useAccount();
+
+	return (
+		data.account?.payouts_enabled &&
+		Boolean( data.account?.settings?.payouts?.schedule?.interval )
+	);
+};
```

Comment:
Using the same checks we were doing in the PHP file, in `includes/class-wc-stripe-account.php`